pages/index.tsx

```diff
@@ -1,71 +1,51 @@
-import Head from 'next/head'
-import Image from 'next/image'
-import styles from '../styles/Home.module.css'
+import Head from "next/head";
+import Header from "../components/Header";
+import Footer from "../components/Footer";
+import Card from "../components/Card";
+
+import styles from "../styles/Home.module.css";
+import styled from "styled-components";
+
+const dummyArray = [
```

Comment:
Please don't call it dummy Array :) 
You will need this Array, even when you change the Data soon, and I feel better when you call it `players`or something like that.